# generated by patch-package 6.4.9
#
# command:
#   npx patch-package a --debug
#
# declared package:
#   a: file:/home/user/src/javascript/patch-package/patch-package-with-pnpm-support/integration-tests/pnpm-workspaces/a-1.0.0.tgz
#
diff --git a/node_modules/a/hello.txt b/node_modules/a/hello.txt
index ce01362..a292c25 100644
--- a/node_modules/a/hello.txt
+++ b/node_modules/a/hello.txt
@@ -1 +1,2 @@
 hello
+asdf
